vignettes/epiparameter.Rmd

```diff
@@ -0,0 +1,521 @@
+---
+title: "Introduction to epiparameter"
+output: rmarkdown::html_vignette
+bibliography: references.bib
+csl: evolution.csl
+vignette: >
+  %\VignetteIndexEntry{epiparameter}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This vignette will use a range of R packages for formatting data and performing a range of calculations. These packages can be thought of as either general data science tools, which will come from the [*tidyverse*](https://www.tidyverse.org/), or tools specific to epidemiological tasks.
+
+```{r setup}
+library(tibble)
+library(epiparameter)
+library(EpiEstim)
+library(EpiNow2)
+library(R0)
+library(outbreaks)
+library(incidence2)
+```
+
+`epiparameter` is an R package that contains a range of delay distributions collated from peer-reviewed reports and meta-analyses, as well as functions to extract the parameters of a right-skewed parametric probability distribution from its percentiles or a median and range. If you would like to contribute to the database on delay distributions in this R package see the [contributing section](#contributing) at the end of this document.
+
+Definitions:
+
+Delay distributions included in `epiparameter`:
+
+-   incubation period: The time delay between the time of infection and the onset of symptoms
+-   onset to admission: The time delay between the time of symptom onset to admission to hospital
+-   onset to death: The time delay between the time of illness onset to death
+-   serial interval: The time delay between the onset of symptoms between a primary case and a secondary case
```

Comment:
The list of implemented delay distributions should have a corresponding `match.arg()` in `list_distributions()` so we return a helpful error message if the user request something epiparameter doesn't provide.

Reply:
Will add `match.arg()` to `list_distributions()` do you also think I should add `match.args()` to the other functions that take string arguments?

Reply:
Added `match.arg()` for functions that take character strings as arguments (3e65f383f54a10826b0d21ab9dbe731112fb3f1c)